.gitignore

```diff
@@ -11,3 +11,7 @@ bin/
 
 # Release artifacts
 release-artifacts/
+
+# Outputs
+*.log
```

Comment:
curious who/what generates *.log from the repo

Reply:
Nothing in the repo generates a *.log, but I like to deploy the image to the cluster and check the logs sometimes so I frequently find myself committing a log file by accident then removing it from the commit.  

Happy to remove this change if you don't think it is necessary, but I figure that if there ever are .log files lying around then probably shouldn't be committed.

Reply:
yah lets rm it